src/components/ec-button-group/ec-button-group.spec.js

```diff
@@ -0,0 +1,218 @@
+import { mount, createLocalVue } from '@vue/test-utils';
+import EcButtonGroup from './ec-button-group.vue';
+import { withMockedConsole } from '../../../tests/utils/console';
+
+function mountButtonGroup(props, mountOpts) {
+  return mount(EcButtonGroup, {
+    propsData: { ...props },
+    ...mountOpts,
+  });
+}
+
+function mountButtonGroupAsTemplate(template, props, mountOpts) {
+  const localVue = createLocalVue();
+
+  const Component = localVue.extend({
+    components: { EcButtonGroup },
+    template,
+  });
+
+  return mount(Component, {
+    localVue,
+    propsData: { ...props },
+    ...mountOpts,
+  });
+}
+
+describe('EcButtonGroup', () => {
+  describe(':props', () => {
+    it(':items - should render the button group with two items all on outline', () => {
+      const wrapper = mountButtonGroup({
+        items: [
+          { text: 'Yes', value: 'yes' },
+          { text: 'No', value: 'no' },
+        ],
+      });
+
+      expect(wrapper.element).toMatchSnapshot();
+    });
+
+    it(':items - should render the button group with two items with one disabled', () => {
+      const wrapper = mountButtonGroup({
+        items: [
+          { text: 'Yes', value: 'yes' },
+          { text: 'No', value: 'no', disabled: true },
+        ],
+      });
+
+      expect(wrapper.element).toMatchSnapshot();
+    });
+
+    it(':items - should render the button group with three items with one disabled', () => {
```

Comment:
honest question: what value does this add to the previous test? We could go on an on "four items and one disabled, five items and one disabled, ...)

Reply:
I cannot see anything you are right, I will remove it